src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/controllers/PreSubmitCallbackController.java

```diff
@@ -124,6 +124,13 @@ public PreSubmitCallbackController(
         return performStageRequest(PreSubmitCallbackStage.ABOUT_TO_SUBMIT, callback);
     }
 
+    @PostMapping(path = "/ccdMidEvent")
```

Comment:
We should add swagger api docs for this new method.  Happy to do that as a separate task but it should be there since we are going to public beta very soon.

Reply:
Thanks ravi. I will add the documentation.